### ISSUE TITLE
onAccessibilityTap :  Property works btut typescript does not recognize that Button should have support for it

### ISSUE THREAD

**Issue by iamAbhi-916** [microsoft/react-native-windows#14856]
override button.d.ts in react native windows : vnext\src-win button and add onAccessibilityTap  declaration as it's a windows only prop , no changes needed upstream.

<img width="1060" height="365" alt="Image" src="https://github.com/user-attachments/assets/a1a23aa2-058f-407b-a7ef-7e594f5981c7" />

**Comment by iamAbhi-916** [comment]
After this is completed, we need to import button from react native windows in react native gallery NewArch\src\examples\ModalExamplePage.tsx

**Comment by vivekkhare31** [comment]
Suggestion: make changes upstream

**Comment by iamAbhi-916** [comment]
@vivekkhare31 this is a window only prop so need to be done only on react native windows
